ClickHouse.Client/Types/PlainDataType.cs

```diff
@@ -17,5 +23,60 @@ public PlainDataType(ClickHouseTypeCode dataType)
         public override ClickHouseTypeCode TypeCode => dataType;
 
         public override string ToString() => TypeCode.ToString();
+        
+        public override string ToHttpParameter(object value)
```

Comment:
This looks like it should not be part of type system, but a separate class like 'HttpFormatter' in `Formats`

Reply:
done